apps/solana_app/solana_sign_txn.c

```diff
@@ -448,9 +451,7 @@ static bool send_signature(solana_query_t *query,
          sizeof(solana_sign_txn_signature_response_t));
 
   solana_send_result(&result);
```

Comment:
can we split this to sign & send functions separately? that way, the secret will be inside one function unit only. Currently, it's split over fetch_seed and send_signature. refer: https://github.com/Cypherock/x1_wallet_firmware/blob/refactor/core-framework/apps/near_app/near_txn.c#L387-L388

Reply:
Will look into it

Reply:
Will take it up as a follow up change.

Reply:
okay.